### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by dannymichel** [rejuvyesh/rmpv#2]
I mean, rmpv <video location> plays the video but its not scrobbling to trakt.tv

**Comment by rejuvyesh** [comment]
Yeah, haven't come around to updating the package for their new api. Pull requests are welcome though.
